lib/wibox/layout/fixed.lua

```diff
@@ -110,6 +110,14 @@ function fixed:get_children()
     return self.widgets
 end
 
+--- Replace the layout children
+-- @tparam table children A table composed of valid widgets
+function fixed:set_children(children)
+    if not children then return self:reset() end
```

Comment:
I am unsure about this. `set_children` is the opposite of `get_children` and it return a table, not an `unpack`. It would rather inconsistent. As for the check, it doesn't hurt. I am just being nice for users, no?